src/utils/types.js

```diff
@@ -0,0 +1,10 @@
+const type = {
```

Comment:
These aren't usually the kinds of things that people usually split out into separate functions. Really the only useful one is `isObject()` and the one you have here is actually wrong.